### PR TITLE
Feature/lp 42 community list

### PR DESCRIPTION
**Pull request by carlosph816**
Adding changes of default image and changing list in VM

https://devaptivist.atlassian.net/jira/software/projects/LP/boards/4?assignee=63cef557c7e5fd35203b00db%2C712020%3A5d5cf1f4-1e1e-423f-94e4-d02438f49add&selectedIssue=LP-42

Screenshot_20230508_144405

### CODE REVIEW

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt [hunk 1]
```diff
@@ -25,16 +25,13 @@ import com.iteneum.designsystem.utils.TextUtils.TWO
 fun CommunityListView(
     viewModel: CommunityListViewModel = hiltViewModel()
 ) {
-    LaunchedEffect(true){
+    LaunchedEffect(true) {
```
Comment (melissaquezada94) [inline feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt:28]: Do you really need this LauchedEffect?

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt [hunk 2]
```diff
@@ -18,26 +18,21 @@ import kotlinx.coroutines.launch
...
 
+    /* THIS STATE IS GOING TO BE USED WHEN WE NEED TO DO A BIG CHANGE TO THE VIEW  */
     private val _state: MutableStateFlow<CommunityListState> =
         MutableStateFlow(CommunityListState.Loading)
     val state: StateFlow<CommunityListState> = _state
 
-    private val _dataInfo: MutableList<ItemModel> = mutableStateListOf()
-    val dataInfo: MutableList<ItemModel>
-        get() = _dataInfo
+    var myData: MutableList<ItemModel> = mutableStateListOf()
```
Comment (melissaquezada94) [inline feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt]: Could you being more descriptive in the model that should be a domain and in the variable myData, this is useful to wont do a lot of changes in the future

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt (hunk 1 above)
Comment (carlosph816) [reply feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt:28]: Yes we need it for this example :D

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt (hunk 2 above)
Comment (carlosph816) [reply feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt]: Do you mean to put a description for "myData"?

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt [hunk 3]
```diff
@@ -25,16 +25,13 @@ import com.iteneum.designsystem.utils.TextUtils.TWO
...
     }
-    val list = remember {
-        viewModel.dataInfo
-    }
     CommunityListContent(
-        list = list,
+        list = viewModel.myData,
         onClickItem = {
-           //TODO HERE WE ARE GOING TO DECIDE WHAT ACTION TO DO
+            //TODO HERE WE ARE GOING TO DECIDE WHAT ACTION TO DO
```
Comment (irv205) [inline feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt:34]: remove this space

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt (hunk 2 above)
Comment (melissaquezada94) [reply feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt]: I meant that you could be more descriptive with the name of the variable and the model's name

Comment (carlosph816) [reply feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListViewModel.kt]: Done

File: core/data/src/main/java/com/iteneum/AmenitieModel.kt [hunk 4]
```diff
@@ -8,7 +8,7 @@ package com.iteneum
  * @author Carlos Hernandez
  */
 //TODO IT NEEDS TO BE PARCELIZE
-data class ItemModel(
+data class AmenitieModel(
```
Comment (irv205) [inline core/data/src/main/java/com/iteneum/AmenitieModel.kt]: Change the name model to "Amenitie"

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt [hunk 5]
```diff
@@ -25,14 +25,11 @@ import com.iteneum.designsystem.utils.TextUtils.TWO
 fun CommunityListView(
     viewModel: CommunityListViewModel = hiltViewModel()
 ) {
-    LaunchedEffect(true){
+    LaunchedEffect(true) {
```
Comment (irv205) [inline feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt:28]: Delete the space

Comment (carlosph816) [reply feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt:28]: I can't delete it cause is part of the reformat code

File: core/data/src/main/java/com/iteneum/AmenitieModel.kt (hunk 4 above)
Comment (carlosph816) [reply core/data/src/main/java/com/iteneum/AmenitieModel.kt]: Done

File: feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt (hunk 3 above)
Comment (carlosph816) [reply feature/communitylist/src/main/java/com/iteneum/communitylist/presentation/CommunityListView.kt:34]: Done

**Review by irv205 — APPROVED** [review]

**Review by dmroldanch — APPROVED** [review]
Good Job 🥇

**Pull request merged**